css/properties/border-right-color.json

```diff
@@ -0,0 +1,56 @@
+{
+  "css": {
+    "properties": {
+      "border-right-color": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-right-color",
+          "support": {
+            "webview_android": {
+              "version_added": "1"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1",
+              "notes": "Firefox also supports the non-standard <a href='https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-border-right-colors'><code>-moz-border-right-colors</code></a> CSS property that sets the right border to multiple colors."
```

Comment:
This will fail "Use non-localized MDN URL (https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-border-right-colors → https://developer.mozilla.org/docs/Web/CSS/-moz-border-right-colors)."